kernel/packages/shared/session/sagas.ts

```diff
@@ -143,12 +312,12 @@ function* login() {
   }
   yield put(userAuthentified(userId, identity, net))
 
-  yield loginCompleted
+  loginCompleted.resolve()
```

Comment:
Just a sanity check: `yield loginCompleted` awaited for the background login to finish... I think.

This completely changes that behaviour to marking the login completed when this line is hit.

Is this the expected behaviour?

We have a couple of `loginCompleted.resolve()` above

Reply:
Yes, i think so. Let me check why it changed and i'll talk you later